dash_core_components/package.json

```diff
@@ -13,7 +13,10 @@
     "publish-all": "node scripts/publish.js",
     "start": "webpack-serve ./webpack.serve.config.js --open",
     "lint": "eslint src",
-    "test": "python -m unittest test.test_integration",
+    "lint:py": "flake8 --ignore=E501,F401,F841,F811 test",
+    "test": "run-s -c test-unit test:py test:pyimport lint format:test lint:py",
+    "test:py": "python -m unittest test.test_integration",
+    "test:pyimport": "python -m unittest test.test_dash_import",
```

Comment:
Can combine them with `python -m unittest discover`

Reply:
A good to know! But `test_dash_import` is funny, since it rewrites files (specifically, making a `dash.py` in the test folder) you can't run it in conjunction with any of the other tests. So if we wanted to use `discover` we'd have to make `test_dash_import` undiscoverable one way or another, and still run it separately.

Reply:
Hmm, that test is strange, not sure we really need to test that.

Reply:
I guess someone put that error message in the code, so they wanted to see it in a test. But yeah, not the most useful test to be lugging around.